pkg/operator/k8sutil/node.go

```diff
@@ -246,3 +247,41 @@ func NodeIsInRookNodeList(targetNodeName string, rookNodes []rookalpha.Node) boo
 	}
 	return false
 }
+
+// AddNodeAffinity will return a v1.NodeAffinity object
+func AddNodeAffinity(nodeAffinity string) *v1.NodeAffinity {
+	newNodeAffinity := &v1.NodeAffinity{
+		RequiredDuringSchedulingIgnoredDuringExecution: &v1.NodeSelector{
+			NodeSelectorTerms: []v1.NodeSelectorTerm{
+				{},
+			},
+		},
+	}
+	// Replace all whitespaces in the string 'nodeAffinity'
+	// Retrieve Labels from the string
+	// ';' separates labels
+	nodeAffinity = strings.Replace(nodeAffinity, " ", "", -1)
+	nodeAffinity = strings.Replace(nodeAffinity, ";", " ", -1)
+	nodeLabels := strings.Fields(nodeAffinity)
```

Comment:
Instead of `strings.Fields()` could you use `strings.Split()`? Then you don't need to first replace the delimiter with a space.

Reply:
ack.